cursorless-talon/src/get_grapheme_spoken_form_entries.py

```diff
@@ -37,6 +27,16 @@ def get_grapheme_spoken_form_entries(
 
 
 def get_graphemes_talon_list() -> dict[str, str]:
+    if grapheme_capture_name not in registry.captures:
```

Comment:
After the merge of literal mark this if statement ended up in the wrong place. It should of course be here where we actually read it from the registry.